test/test.go

```diff
@@ -584,7 +584,7 @@ func (h *Helper) Cleanup() {
 
 // ReadManifest returns the manifest in the current directory.
 func (h *Helper) ReadManifest() string {
-	m := filepath.Join(h.pwd(), "manifest.json")
+	m := filepath.Join(h.pwd(), "manifest.toml")
```

Comment:
I don't believe I can use the constant here (in the test pkg) because that would create a circular dependency?